src/relay/tags.go

```diff
@@ -0,0 +1,41 @@
+package relay
+
+import (
+	"encoding/base64"
+	"strconv"
+
+	"github.com/jackc/pgtype"
+	"github.com/warp-contracts/syncer/src/utils/bundlr"
+	"github.com/warp-contracts/syncer/src/utils/model"
+)
+
+func getTags(payload *Payload, source, env string, interaction *model.Interaction, random []byte) (out []bundlr.Tag) {
```

Comment:
The `Contract-Tx-Id` and `Tx-Id` tags should also be set in this function. For both L1 and L2.

Reply:
Done